cmake/Jemalloc.cmake

```diff
@@ -21,7 +21,7 @@ if(JEMALLOC_INCLUDE_DIR AND JEMALLOC AND JEMALLOC_PIC)
   set_target_properties(im_jemalloc_pic PROPERTIES IMPORTED_LOCATION "${JEMALLOC_PIC}")
   set_target_properties(im_jemalloc PROPERTIES IMPORTED_LOCATION "${JEMALLOC}")
   target_include_directories(jemalloc INTERFACE "${JEMALLOC_INCLUDE_DIR}")
-  target_link_libraries(jemalloc INTERFACE im_jemalloc im_jemalloc_pic)
+  target_link_libraries(jemalloc INTERFACE im_jemalloc_pic im_jemalloc)
```

Comment:
Should we add a comment explaining why the order is important?

Reply:
bold of you to assume I understand this fully :) 

I will add a comment